src/lib/libraries/extensions/index.jsx

```diff
@@ -153,6 +158,20 @@ export default [
             />
         ),
         featured: true,
-        disabled: true
+        disabled: true,
+        launchDeviceConnectionFlow: true,
+        useDeviceList: false,
```

Comment:
Since we settled on using the term `autoScan` for the components, maybe we should use `useAutoScan: true` here instead of the device list?